.travis.yml

```diff
@@ -10,13 +10,53 @@ android:
   - android-26
 env:
 - GRADLE_OPTS="-XX:MaxPermSize=2048M"
+stages:
+  - name: test
+  - name: deploy
+    if: tag IS present
 jobs:
   include:
-  - script: make test
-  - script: make travis
-    if: tag IS present
+    - stage: test
+      script: make test
+  
+    - stage: deploy
+    # flavors sorted alphabetically with three per job
+      script:
+      - fastlane deploy flavor:unbranded
+      - fastlane deploy flavor:medicmobiledemo
+      - fastlane deploy flavor:bracuganda
+    - script:
+      - fastlane deploy flavor:cic_guatemala
+      - fastlane deploy flavor:ebpp_indonesia
+      - fastlane deploy flavor:hope_through_health
+    - script:
+      - fastlane deploy flavor:livinggoods
+      - fastlane deploy flavor:livinggoodskenya
+      - fastlane deploy flavor:livinggoods_assisted_networks
+    - script:
+      - fastlane deploy flavor:livinggoods_innovation_ke
+      - fastlane deploy flavor:livinggoods_innovation_ke_supervisor 
+      - fastlane deploy flavor:livinggoods_innovation_ke_hivst
+    - script:
+      # - fastlane deploy flavor:moh_kenya_siaya_white
```

Comment:
Not white?

Reply:
It isn't actually on the play store yet. Unsure of the story, but we are building it. I've messaged the Siaya team and if we don't need it anymore I'll remove it.